apps/atomy-q/API/app/Adapters/QuotationIntelligence/ProviderQuoteContentProcessor.php

```
@@ -0,0 +1,173 @@
+            if ($score > $bestScore) {
+                $bestScore = $score;
+                $bestItem = $candidate;
+            }
+        }
+
+        if ($bestItem !== null && $bestScore > 0) {
+            return $bestItem;
+        }
+
+        return $rfqLineItems[$index] ?? $rfqLineItems[0] ?? null;
+    }
```

PHPMD (2.15.0)

[warning] The method matchRfqLineItem() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)